src/mmrelay/meshtastic_utils.py

```
@@ -952,35 +1496,107 @@ def _maybe_reset_ble_executor(ble_address: str, timeout_count: int) -> None:
+            _ble_future_started_at = None
+            _ble_future_timeout_secs = None
+            with _ble_timeout_lock:
+                _ble_timeout_counts[ble_address] = 0
+
+        if ble_address in _ble_executor_degraded_addresses:
+            if ble_future_to_cancel is not None:
+                ble_future_to_cancel.cancel()
+                try:
+                    ble_future_to_cancel.result(timeout=0.2)
+                except Exception:
+                    pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)